app/src/main/java/io/pslab/activity/PowerSourceActivity.java

```diff
@@ -29,8 +33,8 @@
 import com.sdsmdg.harjot.crollerTest.Croller;
 import com.sdsmdg.harjot.crollerTest.OnCrollerChangeListener;
 
-import org.apache.commons.lang3.math.NumberUtils;
```

Comment:
Yes, he did! The method  NumberUtils.isCreatable(String) from the library he used did a good job preventing the crash in the Activity, but I noticed that it does not take the locale into account which lead to numbers not being parsed on my test device. This caused the values to be reset on orientation change when the PSLab is not connected which I did not notice until I added the onSaveInstanceState method.

Reply:
@marcnause @CloudyPadmal Yes, this is a perfectly right approach to avoid locales not getting parsed. However, I just had one or points to mention about this. I'll just send these in a review.

Reply:
Excellent!